gradle/libs.versions.toml

```diff
@@ -77,7 +77,7 @@ randomizedrunner  = "2.7.1"
 junit             = "4.13.2"
 junit5            = "6.0.0"
 hamcrest          = "2.1"
-mockito           = "5.16.1"
+mockito           = "5.20.0"
```

Comment:
@andrross Do we need to add this dependency bump in changelog as well?

Reply:
I don't think it's that important